app.js

```diff
@@ -41,6 +41,51 @@ Ext.application({
         Ext.Viewport.add(Ext.create('WhatsFresh.view.Info'));
         Ext.Viewport.add(Ext.create('WhatsFresh.view.Specific'));
 
+        // This is used to iplement android back button
+        if(Ext.os.is('Android')){
+            document.addEventListener("backButton", Ext.bind(onBackKeyDown, this), false);
+            function onBackKeyDown(eve){
+                eve.preventDefault();
```

Comment:
"eve" === event?

Please don't abbreviate your variable names when the explicit noun is a reasonable choice. It doesn't save any resources in the compiling process (javascript is not actually compiled) and it can confuse the meaning of the code you write.